CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.1.4
+
+<!-- generated comparing v6.1.3..master -->
+
+_Oct 15, 2024_
+
+A big thanks to the 9 contributors who made this release possible.
+
+### `@mui/material@6.1.4`
+
+- [Checkbox] Fix disableRipple regression (#44099) @siriwatknp
+- [IconButton] Fix disableRipple behavior when disableRipple is set in MuiButtonBase theme (#43714) @sai6855
+- Make the palette always return new light and dark object (#44059) @siriwatknp
+- Simplify createPalette (#44075) @oliviertassinari
+
+### Docs
+
+- Improve Toolpad Core docs (#43796) @bharatkashyap
+- Tweak Joy UI section in README (#44103) @aarongarciah
+- Fix 404 link to Next.js @oliviertassinari
```

Comment:
Oh interesting, this doesn't feel that great in https://github.com/mui/material-ui/releases/tag/v6.1.4. Fixing that with #44115.